### ISSUE TITLE
Unable to play any song using Chrome

### ISSUE THREAD

**Issue by Netherquark** [icidasset/diffuse#249]
I am using Dropbox to store playlists, etc
Along with GDrive for actual music

[https://youtu.be/d0Tjcb6vc4g](url)
This is the song I used (downloaded using 4K Video Downloader)

My rig:
Windows 10  Pro x64 20H2
AMD 2200G on TUF Gaming B450, 8GB DDR4 2933MHz
Latest Chrome build

I get this error:
![image](https://user-images.githubusercontent.com/65618735/99050286-f686ad00-25bd-11eb-9b13-d064b4497d1e.png)

Followed by this:
![image](https://user-images.githubusercontent.com/65618735/99051680-7a409980-25be-11eb-90d1-e9af48fd2f43.png)

Its the only song in my drive

I was able to replicate the same issues using the following:
Xiaomi Redmi Note 9 Pro (curtana)
On Pixel Experience 10 Plus
Latest Chrome build
The same thing happens whether using desktop mode or not on the phone

PS. It works perfectly on the native windows app though.
PPS. Could you please create a native diffuse app for android? It will surely reduce most of the hassle here.

**Comment by icidasset** [comment]
Thanks for creating the issue!
How did you export the youtube video? MP3, M4A, OGG, or MP4?
(the actual file would be more useful, but not necessary)

**Comment by Netherquark** [comment]
i exported it in mp3 form at 128kbps
its attached as a zip
[Hardwell & DallasK - Area 51 (OUT NOW!) #UnitedWeAre.zip](https://github.com/icidasset/diffuse/files/5540286/Hardwell.DallasK.-.Area.51.OUT.NOW.UnitedWeAre.zip)

**Comment by icidasset** [comment]
Thanks! 🙏 
Hmm yeah, latest Chrome seems to have issues, weird.
Can you try https://nightly.diffuse.sh ? That version seems to work for me.
I'll update https://diffuse.sh soon with v3 (ie. nightly) that should fix this.

**Comment by icidasset** [comment]
> PPS. Could you please create a native diffuse app for android? It will surely reduce most of the hassle here

I don't have an android device to test on. That said, I probably won't do native mobile apps, considering that PWAs (ie. add website to home screen) are getting more popular these days (although it's dissapointing on iOS). I'm even considering getting rid of the "native" desktop apps. It's less for me to maintain and people can easily install it as an app from Chrome/Edge, and soon Firefox.

**Comment by icidasset** [comment]
Made a beta release of v3 in case you want the "native" app:
https://github.com/icidasset/diffuse/releases/tag/3.0.0-beta-1

I think I'll keep creating these "native" wrapper apps until PWAs are better supported on desktop. PWAs on Chrome & Edge are pretty nice, except that it starts Chrome along with your PWA 🤔 Waiting on Firefox and Safari.

**Comment by Netherquark** [comment]
THANK YOU SO MUCH 💘
IT WORKS PERFECTLY 
NO MORE SPOTIFY ADS!!!!!!!!!!!

**Comment by Netherquark** [comment]
> > PPS. Could you please create a native diffuse app for android? It will surely reduce most of the hassle here
> 
> I don't have an android device to test on. That said, I probably won't do native mobile apps, considering that PWAs (ie. add website to home screen) are getting more popular these days (although it's dissapointing on iOS). I'm even considering getting rid of the "native" desktop apps. It's less for me to maintain and people can easily install it as an app from Chrome/Edge, and soon Firefox.

the PWA is pretty good I just wanted an easily accessible icon and the popup was perfect
